core/main.go

```diff
@@ -125,6 +128,72 @@ func (c *Core) ConnectToDatabase() {
 	log.Debugf("connected successfully to database!")
 }
 
+func (c *Core) InitializePrometheus() error {
+	tenants, err := c.db.GetAllTenants(nil)
+	if err != nil {
+		log.Errorf("Error initializing prometheus exporter: %s", err)
```

Comment:
No need to `log.Errorf` if we're going to bail, because `Terminate` will `Alertf`

Reply:
Make these changes for all instances.